### PR TITLE
[Docs] Translate contributing documentation

### PR DESCRIPTION
**Pull request by zhouzaida**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Translate `docs/communtiy/constributing.md`

## Modification

Add `docs_zh_CN/communtiy/constributing.md`

## BC-breaking (Optional)

No

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

**Before PR**:

- [x] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmcv/blob/master/CONTRIBUTING.md) are used to fix the potential lint issues.
- [x] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [x] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [x] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [x] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects, like MMDet or MMCls.
- [x] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs_zh_CN/community/contributing.md [hunk 1]
```diff
@@ -1,3 +1,68 @@
...
+rvm autolibs disable
+
+# install ruby
+rvm install 2.7.1
+```
+
+或者参考 [这个算法库](https://github.com/innerlee/setup) 和 [`zzruby.sh`](https://github.com/innerlee/setup/blob/master/zzruby.sh)。
+After this on every commit check code linters and formatter will be enforced.
+
+>Before you create a PR, make sure that your code lints and is formatted by yapf.
```
Comment (ZwwWayne) [inline docs_zh_CN/community/contributing.md:64]: should  this line be translated?

Comment (zhouzaida) [reply docs_zh_CN/community/contributing.md:64]: yet

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,8 +2,9 @@
 
 All kinds of contributions are welcome, including but not limited to the following.
 
-- Fixes (typo, bugs)
-- New features and components
+- Fixes typo or bugs
```
Comment (ZwwWayne) [inline CONTRIBUTING.md:5]: Fixes -> Fix

File: docs_zh_CN/community/contributing.md [hunk 3]
```diff
@@ -1,3 +1,69 @@
...
+# install rvm
+curl -L https://get.rvm.io | bash -s -- --autolibs=read-fail
+[[ -s "$HOME/.rvm/scripts/rvm" ]] && source "$HOME/.rvm/scripts/rvm"
+rvm autolibs disable
+
+# install ruby
+rvm install 2.7.1
+```
+
+或者参考 [这个算法库](https://github.com/innerlee/setup) 和 [`zzruby.sh`](https://github.com/innerlee/setup/blob/master/zzruby.sh)。
```
Comment (ZwwWayne) [inline docs_zh_CN/community/contributing.md:61]: https://github.com/innerlee/setup is not algorithm library，‘代码库’ might be better

File: CONTRIBUTING.md (hunk 2 above)
Comment (zhouzaida) [reply CONTRIBUTING.md]: done

File: docs_zh_CN/community/contributing.md (hunk 3 above)
Comment (zhouzaida) [reply docs_zh_CN/community/contributing.md]: done

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**